src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -266,6 +266,17 @@ public static String formatDateTimeForInstructorHomePage(Date date) {
         return sdf.format(date);
     }
 
+    /**
+     * Formats {@code dateInUtc} according to the ISO8601 format.
+     */
+    public static String formatDateToIso8601Utc(Date dateInUtc) {
+        if (dateInUtc == null) {
+            return "";
+        }
+        SimpleDateFormat sdf = new SimpleDateFormat(Const.TIME_FORMAT_ISO_8601_UTC);
+        return sdf.format(dateInUtc);
```

Comment:
I found the issue! You need to do `sdf.setTimeZone(SystemParams.TIME_ZONE);` for it to work without the `user.timezone` VM flag.